models/articles.model.js

```diff
@@ -10,14 +10,23 @@ exports.fetchArticleById = (id) => {
     })
 }
 
-exports.fetchArticles = () => {
-    const query = `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.article_id) AS comment_count 
+exports.fetchArticles = (topic) => {
+    const allowedTopics = ['mitch', 'cats', 'paper']
```

Comment:
You may not want to hard code this here as it could cause issues in the future if new topics are added.